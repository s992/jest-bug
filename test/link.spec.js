const link = require("../src/link");

describe("link", () => {

	it("should diff the snapshot", () => {
		expect( link("www.google.com", "google") ).toMatchSnapshot();
	});

});