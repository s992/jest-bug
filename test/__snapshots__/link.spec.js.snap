exports[`link should diff the snapshot 1`] = `"<a href=\"www.google.com\">google</a>"`;
